meson.build

```diff
@@ -110,7 +110,7 @@ if scdoc.found()
       input: filename,
       output: output,
       command: [
-        sh, '-c', '@0@ < @INPUT@ > @1@'.format(scdoc_prog.path(), output)
+        sh, '-c', '@0@ < @INPUT@ > @1@'.format(scdoc_prog.full_path(), output)
```

Comment:
With Meson's new `feed: true` option, we no longer need to use sh to generate man pages. But maybe that's too new for Meson 0.58.

Reply:
Note that sh is a lot faster than running a meson.py wrapper. ;)

Reply:
Does that really make a noticeable difference, though?

Reply:
0.3 seconds for the python version, 0.002 seconds for the shell version.

After evicting the pagecache via /proc/sys/vm/drop_caches, the python version takes 20 seconds, the shell version takes 1 second.

After evicting the Meson source code and the scdoc binary, and *only* those two things, the python version takes 4.5 seconds, and scdoc takes 0.05 seconds.

Most likely real-life performance? Probably somewhere in between -- who knows. In the best-case scenario after you've recently run Meson, the python version noticeably takes time, but barely at all, while in contrast the shell version is... literally instant. In the worst-case scenario reloading python and/or the meson codebase causes noticeable lag on the scale of multiple seconds.

Reply:
Very interesting, thanks!